docs/apache-airflow/concepts/timetable.rst

```diff
@@ -55,15 +55,25 @@ Built In Timetables
 Airflow comes with several common timetables built in to cover the most common use cases. Additional timetables
 may be available in plugins.
 
-CronDataIntervalTimetable
+CronTriggerTimetable
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Set schedule based on a cron expression. Can be selected by providing a string that is a valid
-cron expression to the ``schedule_interval`` parameter of a DAG as described in the :doc:`/concepts/dags` documentation.
+Another timetable which accepts a cron expression. It behaves more compliant with standard cron than ``CronDataIntervalTimetable``.
+
+The main difference from ``CronDataIntervalTimetable`` is that a first DAG Run is kicked off at the start of the period like
+a normal cron, while a first DAG Run of ``CronDataIntervalTimetable`` starts immediately after the DAG is registered.
+
+Note that this timetable does not care the idea of *data interval*. It means the value of ``data_interval_start``,
+``data_interval_end`` and legacy ``execution_date`` are the same - the time when a DAG run is triggered.
```

Comment:
We should also add a couple of paragraphs under ``CronDataIntervalTimetable`` to describe the same thing. It is probably a good idea to add hyperhink on CronDataIntervalTimetable here (and CronTriggerTimetable in the other section) so the user can easily jump between these sections. (See Sphinx documentation on how you can use `:ref:` to achieve this.)

Reply:
+1. Achieved the idea by https://github.com/apache/airflow/pull/23662/commits/ee271b51d53b2da381bc38dd5d709df1c9046757